.github/workflows/CI.yml

```diff
@@ -101,7 +101,7 @@ jobs:
 
       - name: Install package
         run: |
-          conda remove --force openff-toolkit openff-toolkit-base
+          # conda remove --force openff-toolkit openff-toolkit-base
```

Comment:
(not blocking) I'm not sure if this is necessary, and it will be a headache if we forget about it later. Could you see if this can be reintroduced?

Reply:
Turns out it **is** necessary to have this commented out _while Interchange is being installed with `pip`_, since `openff-toolkit-base` is not there to be uninstalled:

```
Run conda remove --force openff-toolkit openff-toolkit-base

PackagesNotFoundError: The following packages are missing from the target environment:
  - openff-toolkit
  - openff-toolkit-base
```

I'll note this more clearly in hopes we notice it later. I expect it'll cause problems and get out attention that way, though.